@types/pal/input.d.ts

```diff
@@ -73,4 +202,173 @@ declare module 'pal/input' {
          */
         public on (eventType: import('cocos/input/types/event-enum').InputEventType, callback: AccelerometerCallback, target?: any);
     }
+
+    type HandleCallback = (res: import('cocos/input/types').EventHandle) => void;
+    /**
+     * Class designed for handle input.
+     */
+    export class HandleInputDevice {
+        /**
+         * Register the handle event callback.
+         */
+        public on (eventType: import('cocos/input/types/event-enum').InputEventType, callback: HandleCallback, target?: any);
```

Comment:
```suggestion
        /**
         * Register the handle event callback.
         * @engineInternal
         */
        public _on (eventType: import('cocos/input/types/event-enum').InputEventType, callback: HandleCallback, target?: any);
```

`HandleInputDevice` is a public interface.
I think the `on` method should be private 
please add the  `@engineInternal` tag in the comment and rename it as `_on`